source/administration/telemetry.rst

```diff
@@ -79,19 +79,19 @@ Channel Moderation Configuration Information (Enterprise Edition Only)
   Information related to channel moderation, including number of channel schemes, number of channels with posting messages disabled for users or guests, number of channels with emoji reactions disabled for users or guests, number of channels with managing members disabled, number of channels with channel mentions disabled for users or guests.
 
 Groups Configuration Information (Enterprise Edition Only)
-  Information related to AD/LDAP groups, including number of groups synced to Mattermost, teams and channels associated to groups, teams and channels synced with groups, and number of group members
+  Information related to AD/LDAP groups, including number of groups synced to Mattermost, teams and channels associated to groups, teams and channels synced with groups, and number of group members.
 
 Plugin Configuration Information
-  Basic information including number of active and inactive plugins, which are using webapp or backend portions, and which `whitelisted Mattermost plugins <https://github.com/mattermost/mattermost-server/blob/master/app/diagnostics.go#L668>`_ are enabled along with their versions.  Some plugins may send summary data such as number of authenticated users of the plugin. 
+  Basic information including number of active and inactive plugins, which are using webapp or backend portions, and which `Mattermost plugins <https://github.com/mattermost/mattermost-server/blob/master/app/diagnostics.go#L668>`_ are enabled along with their versions. The list of plugins is obtained from the Marketplace. If the Marketplace can't be reached, the list of known plugins is used instead. Some plugins may send summary data such as number of authenticated users of the plugin.
```

Comment:
```suggestion
  Basic information including number of active and inactive plugins, which are using webapp or backend portions, and which `Mattermost plugins <https://github.com/mattermost/mattermost-server/blob/master/app/diagnostics.go#L668>`_ are enabled along with their versions. Some plugins may send summary data such as number of authenticated users of the plugin. From version 5.26, the list of plugins is obtained from the Marketplace. If the Marketplace can't be reached, the list of known plugins is used instead.
```

Reply:
Good call, @hanzei. I've added that and reorganized the sentence so it's obvious what the change is. LMK what you think.

Reply:
:+1: from my side